pubsub/rabbitmq/metadata.go

```diff
@@ -203,6 +218,11 @@ func createMetadata(pubSubMetadata pubsub.Metadata, log logger.Logger) (*metadat
 		result.defaultQueueTTL = &ttl
 	}
 
+	result.TLSProperties, err = pubsub.TLS(pubSubMetadata.Properties)
+	if err != nil {
+		return &result, fmt.Errorf("%s invalid TLS configuration: %w", errorMessagePrefix, err)
+	}
+
```

Comment:
If the protocol is AMQPS, check if TLS Properties are provided. If not, return the error info to inform users that the tls config is missed for amqps protocol.

Reply:
Based on https://www.rabbitmq.com/ssl.html#enabling-tls, if rabbitmq server configure `ssl_options.verify` as `verify_none` or `ssl_options.fail_if_no_peer_cert` as `false`, the client is allowed to use amqps protocol without certificates.